core/include/kernel/mutex.h

```diff
@@ -95,10 +95,26 @@ void condvar_broadcast_debug(struct condvar *cv, const char *fname, int lineno);
 void condvar_wait_debug(struct condvar *cv, struct mutex *m,
 			const char *fname, int lineno);
 #define condvar_wait(cv, m) condvar_wait_debug((cv), (m), __FILE__, __LINE__)
+
+/*
+ * Return TEE_ERROR_TIMEOUT If the normal world returns before
```

Comment:
**s/If/if/** + add a period `.` at the end of the sentence (next line).

Ditto at lines 113/114.

Reply:
Updated.